src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1341,6 +1341,64 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr artifact-streaming'] = """
+type: group
+short-summary: Manage artifact streaming in any repositories or images.
+"""
+
+helps['acr artifact-streaming operation'] = """
+type: group
+short-summary: Allows you to manage the streaming artifact creation.
+"""
+
+helps['acr artifact-streaming operation cancel'] = """
+type: command
+short-summary: Cancels the given operation.
+examples:
+  - name: Cancel the streaming artifact creation associated with the id 'OperationId' under repository 'MyRepo' in ACR 'MyRegistry'.
+    text: >
+        az acr artifact-streaming operation cancel -n MyRegistry --repository MyRepo --id OperationId
+"""
+
+helps['acr artifact-streaming operation show'] = """
+type: command
+short-summary: Check the operation status.
+examples:
+  - name: Get the streaming artifact creation status for id 'OperationId' under repository 'MyRepo' in ACR 'MyRegistry'.
+    text: >
+        az acr artifact-streaming operation show -n MyRegistry --repository MyRepo --id OperationId
+"""
+
+helps['acr artifact-streaming create'] = """
+type: command
+short-summary: Create a referrers streaming artifact for a specific image.
+examples:
+  - name: Create the streaming artifact of 'MyImage' in the registry 'MyRegistry'.
+    text: >
+        az acr artifact-streaming create -n MyRegistry -t MyImage
+"""
+
+helps['acr artifact-streaming update'] = """
+type: command
+short-summary: Allows you to enable or disable auto-creation for streaming artifacts for all newer images under a given repository.
```

Comment:
Can you add a clarification to call out after enabling, the new images pushed to the repository will be converted to a new streaming artifact and associated to the original one.

Reply:
Change to: Allows you to enable or disable auto-creation of streaming artifacts for newly pushed images under a given registry. Note: Whenever the flag is enabled, new images pushed into the selected registry will have auto-creation enabled and will be associated with the original image as a referrers artifact.